app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,12 @@
+<div id="<%= dom_id category %>">
```

Comment:
- [ ] Dear @HunterX-7, great job I personally like your design 💯 👌🏻  However it's one of the project requiement we follow the provided design (width, font, size, height, background, color...)  Check these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
* Colors
* Typography: font face, size, and weight.
* Layout: composition and space between elements.
- [ ] Kindly follow the provided design:
 -  You need to change your `ADD CATEGORY ` height, color, and background color, ...  and add this white border around the button text. 👍🏻 
 - The NEW TRANSACTION button has a similar problem. Kindly fix it as well. 💯  👍🏻

Your Implementation | Original Design
:---:|:---:
![image](https://github.com/blueberry1312/rails-capstone/assets/79058364/f556f887-ad0a-42ae-848b-1f68e671f8fa) | ![image](https://user-images.githubusercontent.com/79058364/219786288-a43c8eb4-d309-4b09-9b4e-02b929ccd7fb.png)